lib/actions/deleteObject.js

```diff
@@ -1,31 +1,94 @@
 const { messages } = require('elasticio-node');
 const MetaLoader = require('../helpers/metaLoader');
 const sfConnection = require('../helpers/sfConnection.js');
+const helpers  = require('../helpers/deleteObjectHelpers.js')
 
+/**
+ * Des: Taken from lookupObject.js due to overlapping functionality
+*/
+module.exports.objectTypes = function objectTypes(configuration) {
+  const metaLoader = new MetaLoader(configuration, this);
+  return metaLoader.getSearchableObjectTypes();
```

Comment:
It seems that we limited to some object types. This action will be backward incompatible with current iteration? CC: @jhorbulyk

Reply:
This has been addressed and presented to Jacob. Previous flows should be compatible with this edition.